book/02-git-basics/sections/viewing-history.asc

```diff
@@ -284,7 +284,7 @@ For example, if you want to see which commits modifying test files in the Git so
 
 [source,console]
 ----
-$ git log --pretty="%h - %s" --author=gitster --since="2008-10-01" \
+$ git log --pretty="%h - %s" --author=Junio Hamano --since="2008-10-01" \
```

Comment:
Probably this should be `--author='Junia Hamano'`.

Reply:
It works without quotes; at least for me :)

Reply:
```
$ git log --pretty="%h - %s" --author=Junio Hamano --since="2008-10-01"
fatal: ambiguous argument 'Hamano': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

In any case, even if it works in some places, we should follow good practices and wrap arguments in quotes.

Also:

```
$ git log --pretty="%h - %s" --author='Junio Hamano' --since="2008-10-01" | wc -l
       2
```

Looks like Junio signs his commits as `Junio C Hamano` most of the time. Can you fix that as well?